packages/builtin/stdin.pony

```diff
@@ -30,19 +30,34 @@ interface tag DisposableActor
   """
   be dispose()
 
+interface tag InputStream
+  """
+  Asynchronous access to some input stream.
+  """
+  be apply(notify: (InputNotify iso | None), chunk_size: USize = 32)
+    """
+    Set the notifier. Optionally, also sets the chunk size, dictating the
+    maximum number of bytes of each chunk that will be passed to the notifier.
+    """
+
+  be dispose()
```

Comment:
Maybe this one could be given a default implementation of `=> None`?

This would make it easier to implement the `InputStream` interface in cases where there is nothing that needs to be disposed. For example, if you wanted to mock program input from a String.

Reply:
Good point. I'll add that.